client/src/main/java/io/pravega/client/BatchClientFactory.java

```diff
@@ -103,4 +103,12 @@ static BatchClientFactory withScope(String scope, ClientConfig config) {
      * @return A list of segment range in between a start and end stream cut.
      */
     List<SegmentRange> getSegmentRangeBetweenStreamCuts(final StreamCut startStreamCut, final StreamCut endStreamCut);
+
+    /**
+     * Provides a nextStreamCut to an approxDistance based on the startingStreamCut.
```

Comment:
May be Provides nearest streamcut in future depending on the distance and current streamcut.

Reply:
Done.